plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -4107,4 +4517,11 @@ protected void verifyVersionedQueryFailurePermissible(Exception e)
                         "Delta Lake snapshot ID does not exists: .*|" +
                         "Unsupported type for table version: .*");
     }
+
+    private void assertTableLocation(String tableName, String expectedLocation)
+    {
+        assertThat(query("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM " + tableName))
+                .skippingTypesCheck()
+                .matches("VALUES '%s'".formatted(expectedLocation));
```

Comment:
nit: We could use `computeScalar` method to avoid skippingTypesCheck method:
```java
        assertThat(computeScalar("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM " + tableName))
                .isEqualTo(expectedLocation);
```

By the way, this method doesn't work for empty tables. That's why we use SHOW CREATE TABLE result in other Delta Lake tests. 
https://github.com/trinodb/trino/blob/ac71d158b5ec33e69da0c4484d659dd5ed144629/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeRegisterTableProcedureTest.java#L357-L367

Reply:
Actually that method was not used and we have replaced with `SHOW CREATE` type of assertion. Removed that method.